.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
.editorconfig supported by many editors (http://editorconfig.org/) by default or with plugin. So maybe we should have it in repo?